README.md

```diff
@@ -97,7 +97,9 @@ To run tests, run the following command:
 
 `Rspec spec`
 
-[LiveSrever](https://budget-o4f5.onrender.com)
+[LiveSrever](https://budget-app-5xj4.onrender.com/)
```

Comment:
- [ ] [REQUIRED] The issue about the project deployment is not resolved. As mentioned by the previous reviewer when  When I tried to sign up I didn't receive any email confirmation instructions. So I can t sign as you can see on the video I'm sharing with you below :arrow_down: . Please make sure that you fix that issue in order to make your app's live demo work as expected because it is very important to have an application work from development to production :+1: 

https://www.loom.com/share/35e444887701435985b1c4012767ba47